examples/plot_DISCO_convolutions.py

```diff
@@ -0,0 +1,200 @@
+"""
+Training a SFNO on the spherical Shallow Water equations
```

Comment:
I think this title and caption are incorrect

Reply:
adressed this in new commit

Reply:
No change is showing